<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogOutcome</class>
 <widget class="QDialog" name="DialogOutcome">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Option Outcome</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Action</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="combo_action">
     <item>
      <property name="text">
       <string>ENEMY_ENCOUNTER</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>LOOT_ROLL</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>EVENT_TRIGGER</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>TEXT</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>BATTLE</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ADD_SURVIVOR</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>REMOVE_SURVIVOR</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_text">
     <property name="text">
      <string>Text</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="text_outcome_text"/>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Chance</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="spinner_chance">
     <property name="maximum">
      <number>100</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_tags">
     <property name="text">
      <string>Tags</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="container_tags"/>
   </item>
   <item>
    <widget class="QLabel" name="label_eventid">
     <property name="text">
      <string>Event ID</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="spinner_eventid">
     <property name="maximum">
      <number>99999999</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Enemies</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="spinner_enemy_id">
     <property name="maximum">
      <number>999</number>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QListWidget" name="list_enemies"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QToolButton" name="button_add_enemy">
         <property name="minimumSize">
          <size>
           <width>25</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>+</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="button_remove_enemy">
         <property name="minimumSize">
          <size>
           <width>25</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogOutcome</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogOutcome</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
